security/pkg/pki/ca/generate_cert.go

```diff
@@ -46,8 +46,11 @@ type CertOptions struct {
 	// like kubernetes service account.
 	Host string
 
-	// The validity bounds of the issued certificate.
-	NotBefore, NotAfter time.Time
+	// The NotBefore field of the issued certificate.
+	NotBefore time.Time
+
+	// TTL of the certificate. NotAfter - NotBefore
+	TTL time.Duration
```

Comment:
Should we give a default value of this?

Reply:
Yes, we do have the default values for the TTLs. The default values are given at the CA and node agent (as flags), this CertOptions is always initialized on CA and node agent.